Library/Formula/sshuttle.rb

```diff
@@ -1,11 +1,11 @@
 require 'formula'
 
 class Sshuttle < Formula
-  homepage 'https://github.com/apenwarr/sshuttle'
-  url 'https://github.com/apenwarr/sshuttle/archive/sshuttle-0.61.tar.gz'
-  sha1 '05551cdc78e866d983470ba4084beb206bacebd8'
+  homepage 'https://github.com/sshuttle/sshuttle'
+  url 'https://github.com/sshuttle/sshuttle/archive/sshuttle-0.71.tar.gz'
+  sha256 '62f0f8be5497c2d0098238c54e881ac001cd84fce442c2506ab6d37aa2f698f0'
 
-  head 'https://github.com/apenwarr/sshuttle.git'
+  head 'https://github.com/sshuttle/sshuttle.git'
 
   def install
```

Comment:
According to the other two PRs, `libexec.install Dir['*']` should be `libexec.install Dir["src/*"]` now?